### ISSUE TITLE
Feedback: Testing new MacVim build with PeepOpen

### ISSUE THREAD

**Issue by topfunky** [topfunky/PeepOpen-Issues#144]
Reply here if you have issues or other feedback for the private MacVim build.

**Comment by tibra** [comment]
Works fine for me so far, but I will do some more testing this week.

**Comment by moutten** [comment]
After a week, this build is working great for me as well.

**Comment by tibra** [comment]
I've had one issue, see below. I had three tabs open in MacVim, all with files from the same rails working directory. When I first opened PeepOpen it raised a spinning beach ball and deep-freezed. When I tried again after hard killing PeepOpen, it worked fine again.

Couldn't Find Files
Try another directory or see the Project Root Pattern preferences in the Advanced tab.

Update: Experienced another freeze while hitting Reload. Documents to track are at 1000.
